### PR TITLE
Add z-index to FormGroupContainer's remove button

### PR DESCRIPTION
**Pull request by jfurrow**
This PR:
* Renames `.form-remove` to `.form-group-container-action-button`
* Gives a z-index to `.form-group-container-action-button` to prevent it from rendering below sibling elements

### CODE REVIEW

File: src/js/components/form/FormGroupContainer.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const FormGroupContainer = (props) => {
   let removeButton = null;
   if (props.onRemove != null) {
     removeButton = (
-      <div className="form-remove">
+      <div className="form-group-container-action-button">
```
Comment (mlunoe) [inline src/js/components/form/FormGroupContainer.js]: Should this be `form-group-actions-container`?

File: src/styles/components/form-elements/styles.less [hunk 2]
```diff
@@ -35,9 +41,20 @@
     position: absolute;
   }
 
-  .form-remove {
+  .form-group-container-action-button {
```
Comment (mlunoe) [inline src/styles/components/form-elements/styles.less]: `form-group-actions-container`?

File: src/styles/variables/variables-z-index.less [hunk 3]
```diff
@@ -5,6 +5,7 @@
 @z-index-dygraph-chart: @z-index-side-panel + 10;
 @z-index-dygraph-hover-label: 100;
 @gemini-scrollbar-handle: @z-index-header-tabs + 1;
+@z-index-form-group-container-remove-button: 1;
```
Comment (mlunoe) [inline src/styles/variables/variables-z-index.less]: Should this be `@z-index-form-group-actions-container`?

File: src/styles/components/form-elements/styles.less (hunk 2 above)
Comment (jfurrow) [inline src/styles/components/form-elements/styles.less]: It's a bit funky to read but it's actually used inside of the `FormGroupContainer` component, so I do think it makes sense... do you still think it should be changed?

Comment (mlunoe) [inline src/styles/components/form-elements/styles.less]: Ah, ok. But it is not on the action button? It's a wrapper around the action button, right?

Comment (jfurrow) [inline src/styles/components/form-elements/styles.less]: That's true, the element with this class is not the actual button. And I could see it containing more than one button.

What do you think about `.form-group-container-action-button-group`? It's a bit wordy but it does make sense to me...

**Comment by philipnrmn** [comment]
I can't actually force the remove button to render under its siblings without these changes, but this looks good to me in any case.

**Comment by orlandohohmeier** [comment]
@mlunoe @jfurrow I've adjusted the class and var names as suggested.
